docs/src/main/asciidoc/_admin-guide/subsystem-configuration/EJB3.adoc

```diff
@@ -174,6 +184,51 @@ This is used to enable remote EJB invocations. It specifies a space-separated li
 connectors to use (as defined in the remoting subsystem configuration),
 and the thread pool to use for remote invocations.
 
+[[profile]]
+=== <profile>
+
+Remote profile specificies a configuration of remote invocations that can
+be referenced by many deployments. EJBs that are meant to be invoked can
+be discovered in either static or dynamic way.
+
+Static discovery decides which remote node to connect based on the information
+provided by the user.
```

Comment:
Perhaps s/user/administrator  ?

AIUI the end user making the request isn't involved, so it's a user who administers the server config.